Polish/kadarka/kadarka.md

```diff
@@ -1,18 +1,22 @@
-To dobra decyzja, bo pragnienie coraz bardziej si� wzmaga. Co z tego, �e nie masz na sobie
-spodni? Kogo to obchodzi, skoro jak widzisz, w tej krainie wszyscy chodz� bez majtek.
-W Biedronce litr dobrej p�s�odkiej Kadarki kosztuje 10 z�. To niewiele, za tak
-dobre wino i przy okazji jest jego tyle, �e mo�esz podzieli� si� z nim z wszystkimi krasnoludkami,
-skrzatami i dobrymi duszkami, kt�re zamieszkuj� poblisk� ��k�, tam gdzie jest uj�cie wody miejskiej.
-W pobliskiej Biedronce kupujesz Kadark�.
+To dobra decyzja, bo pragnienie coraz bardziej się wzmaga. Co z tego, ¿e nie masz na sobie
+spodni? Kogo to obchodzi, skoro jak widzisz, w tej krainie wszyscy chodz± bez majtek.
+W Biedronce litr dobrej półsłodkiej Kadarki kosztuje 10 zł. To niewiele, za tak
+dobre wino i przy okazji jest jego tyle, ¿e mo¿esz podzieliæ się z nim z wszystkimi krasnoludkami,
+skrzatami i dobrymi duszkami, które zamieszkują poblisk± łąkę, tam gdzie jest ujęcie wody miejskiej.
+W pobliskiej Biedronce kupujesz Kadarkę.
 
-Mo�e masz ochot� przej�� si� na poblisk� ��k�?
+Wpada Ci do głowy pomysł, że niekoniecznie chce Ci się iść tak daleko na lake. Może lepszym pomysłem będzie "impreza" za sklepem.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.